src/runtime/node/constants/crypto.ts

```diff
@@ -0,0 +1,58 @@
+export const OPENSSL_VERSION_NUMBER = 269_488_319;
```

Comment:
Honestly, i'm not sure about these hardcoded versions. 

If any library depends on them for a minimal version for specific behavior, we are kinda giving false information to it that _might_ end-up into a chain of issues or possible vulnerabilities. 

I would set SSL versions to `0` to reduce such risks

Reply:
Good catch @pi0. `0` sounds reasonable to me.